guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/repositories/git/GitMetadataStoreImpl.java

```diff
@@ -33,7 +33,7 @@
 
     private Logger logger = LoggerFactory.getLogger(GitMetadataStoreImpl.class);
     public static final String SEPARATOR = "/";
-    public static final String METADATA = "default://system/metadata";
```

Comment:
@porcelli @manstis  I've to change metadata filesystem name due to namespace clash with system

Reply:
@ederign Is this "backwards" compatible with older installations? I don't mind if it recreates a new ```.niogit/metadata``` repository... as long as it doesn't break! @wmedvede IIRC you was asking about why ```metadata``` was under ```.niogit/system/metadata```? IDK if this change affects anything you was doing.. and if not you @wmedvede I wonder who it was!??!